Sources/Formworks/Property Wrappers/AutoLayout.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  File.swift
+//  
+//
+//  Created by Artur Carneiro on 28/09/20.
+//
+
+import UIKit
+
+/// Property wrapper used to make it easier to instantiate a `UIView` or
+/// its subclasses ready for Auto layout. **You should only use this property wrapper if you only
+/// need the object to be initialized with the following init:** `init(frame: .zero)`.
+@propertyWrapper final class AutoLayout<View: UIView> {
```

Comment:
Is `AutoLayout` a good name for this property wrapper?
I don't think it describes well what it's doing and it might confuse those reading the code afterwards as AutoresizingMask can also be considered a form of Auto Layout

Reply:
Maybe something along the lines of `ManualLayout` or `ConstraintBased`?

Reply:
I think `ManualLayout` is a good fit for this. I will push the change once our CI is fixed.